vendor/bat-native-ads/src/bat/ads/internal/ad_events/ad_event_util_unittest.cc

```diff
@@ -0,0 +1,228 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ad_events/ad_event_util.h"
+
+#include <string>
+
+#include "base/guid.h"
+#include "bat/ads/internal/bundle/creative_ad_notification_info.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace ads {
+
+namespace {
+
+CreativeAdNotificationInfo GetCreativeAdNotification() {
+  CreativeAdNotificationInfo creative_ad_notification;
+
+  creative_ad_notification.creative_instance_id = base::GenerateGUID();
+  creative_ad_notification.advertiser_id = base::GenerateGUID();
+
+  return creative_ad_notification;
+}
+
+CreativeAdNotificationInfo GetCreativeAdNotificationForAdvertiser(
+    const std::string& advertiser_id) {
+  CreativeAdNotificationInfo creative_ad_notification;
+
+  creative_ad_notification.creative_instance_id = base::GenerateGUID();
+  creative_ad_notification.advertiser_id = advertiser_id;
+
+  return creative_ad_notification;
+}
```

Comment:
Decouple to a `creative_ad_notification_unittest_util` so they can be used for other tests